app/pods/publications/publication/documents/controller.js

```diff
@@ -227,6 +229,8 @@ export default class PublicationDocumentsController extends Controller {
     };
     this.showLoader = false;
     this.renderPieces = true;
+
+    this.transitionToRoute('publications.publication.translations');
```

Comment:
Is dit een vraag van de gebruikers dat je reroute na aanmaken aanvraag of gebruikt tijdens testen?
Als je meerdere aanvragen wil doen (voor grotere dossier waar ze meerdere kleine publicaties willen doen) is dit mogelijk ongewest gedrag.